end_to_end/signal_handler_test.go

```diff
@@ -11,6 +13,14 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func CreateOutput(cmd *exec.Cmd) (*bufio.Scanner, strings.Builder) {
+	outPile, _ := cmd.StdoutPipe()
+	outScanner := bufio.NewScanner(outPile)
```

Comment:
Why do we need extra buffer here? Why we can't just read from process stdout?

Reply:
We need a scanner to implement the waiting of output.